src/kOS/Suffixed/VesselTarget.cs

```diff
@@ -464,6 +465,13 @@ private void InitializeSuffixes()
             AddSuffix("MESSAGES", new NoArgsSuffix<MessageQueueStructure>(() => GetMessages()));
         }
 
+        public global::Part GetControlPart()
+        {
+            global::Part res = Vessel.GetReferenceTransformPart(); //this can actually be null if "control from here" has never been used
```

Comment:
With crewed pods it may automatically do "control from here" on one of them if nothing is set. 
Try starting with a probe core, then put a crewed module upside down and put a kerbal in - the navball will be upside down (I've got this effect even with the cabin that has no control authority!).

I first made it without this addition and tested on couple crafts and it worked. Then I wrote a script that uses it, ran it on one thing controlled with just a probe core (on which I haven't changed the control point) and got the null reference